app/models/miq_widget.rb

```diff
@@ -513,7 +513,8 @@ def sync_schedule(schedule_info)
       raise _("Unsupported interval '%{interval}'") % {:interval => interval}
     end
 
-    sched = MiqSchedule.create!(
+    sched = MiqSchedule.find_by(:name => description)
```

Comment:
Shouldn't the schedule be updated if found?  Otherwise we can have a broken reference from the schedule back to the widget

Reply:
@bdunne 👍